package.json

```diff
@@ -30,6 +30,7 @@
     "codelyzer": "^5.0.0",
     "css-mqpacker": "^7.0.0",
     "cssnano": "^4.1.10",
+    "fs-extra": "^8.1.0",
```

Comment:
kan ook gewoon een spawn('cp zijn in principe

Reply:
vind het niet echt ok om een dependency te installeren puur alleen voor een copy

Reply:
of dit met util promisify https://nodejs.org/api/fs.html#fs_fs_copyfile_src_dest_flags_callback

Reply:
zit ook al in style.scss